core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -989,7 +989,7 @@ private[spark] class TaskSetManager(
         if (ef.className == classOf[TaskOutputFileAlreadyExistException].getName) {
           // If we can not write to output file in the task, there's no point in trying to
           // re-execute it.
-          logError(log"Task ${MDC(TASK_ID, info.id)} in stage ${MDC(STAGE_ID, taskSet.id)} " +
+          logError(log"Task ${MDC(TASK_INFO_ID, info.id)} in stage ${MDC(STAGE_ID, taskSet.id)} " +
             log"(TID ${MDC(TID, tid)}) can not write to output file: " +
```

Comment:
what's the different between TID and TASK_ID?

Reply:
task info id is `def id: String = s"$index.$attemptNumber"`

Reply:
I'm actually pointing #L993

Reply:
Good point, I unified TID and TASK_ID